scrabble_challenge/scrabble_challenge.py

```diff
@@ -0,0 +1,74 @@
+import sys
+
+CHARACTERS_SCORE = {"a": 1, "c": 3, "b": 3, "e": 1, "d": 2, "g": 2,
+                    "f": 4, "i": 1, "h": 4, "k": 5, "j": 8, "m": 3,
+                    "l": 1, "o": 1, "n": 1, "q": 10, "p": 3, "s": 1,
+                    "r": 1, "u": 1, "t": 1, "w": 4, "v": 4, "y": 4,
+                    "x": 8, "z": 10}
+
+
+def get_word_score(valid_word):
+    """
+    Calculates the scrabble score of valid words and returns it.
+    """
+    word_score = 0
+    for character in valid_word:
+        word_score += CHARACTERS_SCORE[character]
+    return word_score
+
+
+def is_word_from_rack(word, rack):
+    """
+    This function check the word that if it is made of the letters that are subset of rack letters.
+    """
+    rack = rack.lower()
+    for character in word:
+        if character not in rack:
+            return False
+        else:

```

Comment:
This is correct, but logically we don't have to specifically write else here as the if clause have a return statement.